plugins/scaffolder-backend/src/scaffolder/stages/templater/types.ts

```diff
@@ -42,7 +46,7 @@ export type TemplaterRunResult = {
  */
 export type TemplaterRunOptions = {
   directory: string;
-  values: RequiredTemplateValues & Record<string, JsonValue>;
+  values: RequiredTemplateValues & Record<string, any>;
```

Comment:
Tried so hard to avoid this, but couldn't. It kept complaining that  `gitUrlParse.GitUrl` can't be of type `JsonValue` and i even did `JsonValue | gitUrlParse.GitUrl`